diagnostics/diagnostics_metrics.go

```diff
@@ -0,0 +1,95 @@
+package diagnostics
+
+import (
+	"runtime"
+
+	"github.com/stripe/veneur/v14"
+)
+
+var (
+	gitSha                  string = "git_sha: " + veneur.VERSION
+	prevTotalMalloc         uint64 = 0
+	prevTotalFreed          uint64 = 0
+	prevTotalHeapAllocBytes uint64 = 0
+)
+
+func CollectUptimeMetrics(s *veneur.Server) {
+	s.Statsd.Count("uptime_ms", s.Interval.Milliseconds(), []string{"commit_version:" + veneur.VERSION}, 1)
+}
+
+// This function take inspiration from the prometheus golang code base: https://github.com/prometheus/client_golang/blob/24172847e35ba46025c49d90b8846b59eb5d9ead/prometheus/go_collector.go
+func CollectRuntimeMemStats(s *veneur.Server) {
+	var m runtime.MemStats
```

Comment:
nit: It's a bit more readable to have
```
var memstatsCurrent runtime.MemStats
var memstatsPrevious runtime.MemStats
```

and do something like
```
memstatsPrevious = memstatsCurrent
```

Reply:
done